src/changelog.rst

```diff
@@ -18,6 +18,47 @@ with the change that has been applied due to it.
    just the language changes that had an impact to the FLS. See the `release
    notes`_ for a full list of changes.
 
+Language changes in Rust 1.81.0
+-------------------------------
+
+* `Abort on uncaught panics in `extern "C"` functions. <https://github.com/rust-lang/rust/pull/116088/>`_
+
+  * No change: unwinding is not specified in the FLS
+
+* `Fix ambiguous cases of multiple `&` in elided self lifetimes. <https://github.com/rust-lang/rust/pull/117967/>`_
+
+  * Changed paragraph: :p:`fls_crb6m6b3cdwh`
+
+  * New paragraph: :p:`fls_d4u3y82hdadc`
+
+* `Stabilize `#[expect]` for lints (RFC 2383), <https://github.com/rust-lang/rust/pull/120924/>`_ like `#[allow]` with a warning if the lint is _not_ fulfilled.
+
+  * No change: lints are not part of the FLS
```

Comment:
We need to add the attribute to the list in https://spec.ferrocene.dev/attributes.html#built-in-attributes.